--- a/drivers/net/wireless/rockchip_wlan/rkwifi/rk_wifi_config.c	2020-04-15 16:22:51.000000000 +0530
+++ b/drivers/net/wireless/rockchip_wlan/rkwifi/rk_wifi_config.c	2020-04-15 16:25:48.000000000 +0530
@@ -15,9 +15,8 @@
 /*
  * Set Firmware Path
  */
-
-#define VENDOR_ETC_FIRMWARE "/vendor/etc/firmware/"
-#define SYSTEM_ETC_FIRMWARE "/system/etc/firmware/"
+#define VENDOR_ETC_FIRMWARE "/lib/firmware/"
+#define SYSTEM_ETC_FIRMWARE "/lib/firmware/"
 char ANDROID_FW_PATH[64] = {0};

 extern int get_wifi_chip_type(void);
@@ -68,6 +67,11 @@
         sprintf(nvram, "%s%s", ANDROID_FW_PATH, "nvram_ap6212.txt");
 }

+if (chip == WIFI_AP6236) {
+    sprintf(fw, "%s%s", ANDROID_FW_PATH, "fw_bcm43436b0.bin");
+        sprintf(nvram, "%s%s", ANDROID_FW_PATH, "nvram_ap6236.txt");
+}
+
 if (chip == WIFI_AP6234) {
     sprintf(fw, "%s%s", ANDROID_FW_PATH, "fw_bcm43341b0_ag.bin");
 	sprintf(nvram, "%s%s", ANDROID_FW_PATH, "nvram_AP6234.txt");
